scipy/spatial/transform/_rotation.pyx

```diff
@@ -2777,12 +2794,11 @@ cdef class Rotation:
             Object containing the mean of the rotations in the current
             instance.
 
-        Notes
-        -----
-        The mean used is the chordal L2 mean (also called the projected or
-        induced arithmetic mean). If ``p`` is a set of rotations with mean
-        ``m``, then ``m`` is the rotation which minimizes
-        ``(weights[:, None, None] * (p.as_matrix() - m.as_matrix())**2).sum()``.
+        References
+        ----------
+        .. [1] Hartley, Richard, et al.,
```

Comment:
This is the doc recommended when the mean() function was added in https://github.com/scipy/scipy/pull/10866 (and I agree it's the best reference I could find myself)